scripts/us_census/pep/pep_by_asrh/preprocess.py

```diff
@@ -0,0 +1,1072 @@
+    # Creating SV"s name using SV, Age Column
+    data_df["SV"] = data_df.apply(lambda row: _create_sv(row.SV, row.Age),
+                                  axis=1)
+    # Deriving Measurement Method for the SV"s
+    f_cols = ["Year", "Location", "SV", "Measurement_Method", "Count_Person"]
+    data_df = _add_measurement_method(data_df, "SV", "Measurement_Method")
+    return data_df[f_cols]
+
+
+def _process_county_2010_2020(file_path: str) -> pd.DataFrame():
+    """
+    Returns the Cleaned DataFrame consists
+    county data for the year 2010-2020.
+
+    Args:
+        file_path (str): Input File Path
+
+    Returns:
+        pd.DataFrame: Cleaned DataFrame
+    """
+
+    data_df = _load_data_df(path=file_path,
+                            file_format="csv",
+                            header=0,
+                            encoding="latin-1")
+    data_df = data_df[(~data_df["YEAR"].isin([1, 2, 13])) &
+                      (data_df["AGEGRP"] != 0)].reset_index(drop=True)
+    data_df["YEAR"] = data_df["YEAR"].astype("str").str.replace(
+        "14", "13").astype("int")
+    data_df["YEAR"] = 2007 + data_df["YEAR"]
+    # Mapping Dataset Headers to its FullForm
+    cols_mapper = _get_mapper_cols_dict("header_mappers")
+    cols = data_df.columns.to_list()
+    for idx, val in enumerate(cols):
+        cols[idx] = cols_mapper.get(val, val)
+    data_df.columns = cols
+    data_df["STATE"] = data_df["STATE"].astype("str").str.pad(width=2,
+                                                              side="left",
+                                                              fillchar="0")
+    data_df["COUNTY"] = data_df["COUNTY"].astype("str").str.pad(width=3,
+                                                                side="left",
+                                                                fillchar="0")
+    data_df["Location"] = "geoId/" + data_df["STATE"] + data_df["COUNTY"]
+    # Deriving New Columns
+    derived_cols = _get_mapper_cols_dict("county_2010_2020")
+    data_df = _derive_cols(data_df, derived_cols)
+    cols = cols + list(derived_cols.keys())
+    f_cols = [val for val in cols if "Hispanic" in val]
+    data_df["Age"] = data_df["AGEGRP"].apply(_get_age_grp)
+    data_df["Age"] = data_df["Age"].str.replace("85To89", "85")
+    data_df = pd.melt(data_df,
+                      id_vars=["Year", "Location", "Age"],
+                      value_vars=f_cols,
+                      var_name="SV",
+                      value_name="Count_Person")
+    # Creating SV"s name using SV, Age Column
+    data_df["SV"] = data_df.apply(lambda row: _create_sv(row.SV, row.Age),
+                                  axis=1)
+    f_cols = ["Year", "Location", "SV", "Measurement_Method", "Count_Person"]
+    # Deriving Measurement Method for the SV"s
+    data_df = _add_measurement_method(data_df, "SV", "Measurement_Method")
+    return data_df[f_cols]
+
+
+def _derive_nationals(data_df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
nit: it would be useful to move this function to the top. The way I see it, you have the following types of functions:

1. Purely helper functions. These should be on the top of the file. So I would move _derive_nationals() there. I would also move the _generate_mcf() and _generate_tmcf() up at the top too.
2. The _process_*() functions. Those could go next. You could also consider moving all the _process_* functions to a different file for better organization. But that's not necessary.
3. The public functions (which you already have at the bottom).